src/template.mligo

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: 2021 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+// Replace TEMPLATE_H with something unique corresponding to your
+// variant. For example, if your variant is called MyBaseDAOVariant you
+// can change TEMPLATE_H to MYBASEDAOVARIANT_H so this section reads
+//
+// #if !MYBASEDAOVARIANT_H
+// #define MYBASEDAOVARIANT_H
+
+#if !TEMPLATE_H
+#define TEMPLATE_H
```

Comment:
Couldn't this be fixed instead and used by all the implementations (including the ones already existing here)?

Something like:
```suggestion
#if !IMPLEMENTATION
#define IMPLEMENTATION
```
or
```suggestion
#if !VARIANT
#define VARIANT
```
or something like that?

Reply:
Ah, yes. Good catch! Let me try.

Reply:
Have changed.